com.unity.render-pipelines.high-definition/Runtime/PostProcessing/PostProcessSystem.cs

```diff
@@ -1300,6 +1321,7 @@ struct DepthOfFieldParameters
             public BlueNoise.DitheredTextureSet ditheredTextureSet;
 
             public HDCamera camera;
+            public Vector2Int viewport;
```

Comment:
Same here (comment on viewport name above)

Reply:
Renamed it to viewportSize